gcs/iam.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Simulate IAM operations."""
+
+import base64
+import binascii
+import flask
+import json
+
+import testbench
+
+IAM_HANDLER_PATH = "/iamapi"
+iam = flask.Flask(__name__)
+iam.debug = False
+iam.register_error_handler(Exception, testbench.error.RestException.handler)
```

Comment:
Q: Since the registering handlers are moved to their own files, I’m assuming we no longer need them in `emulator.py`?

Reply:
Yes, I think when I move the `emulator.py` file I will remove the additional registrations.  Now that I know how to unit test Flask applications that might get better testing too.